docs/quickstart.rst

```diff
@@ -82,32 +88,38 @@ definitions from
 `scikit-learn <https://scikit-learn.org/stable/modules/classes.html#module-sklearn.metrics>`_
 we can evaluate metrics to get a group summary as below:
 
+.. doctest::
+
     >>> from fairlearn.metrics import group_summary
     >>> from sklearn.metrics import accuracy_score
     >>> from sklearn.tree import DecisionTreeClassifier
     >>> 
     >>> classifier = DecisionTreeClassifier(min_samples_leaf=10, max_depth=4)
     >>> classifier.fit(X, y_true)
-    >>> 
+    DecisionTreeClassifier(...)
     >>> y_pred = classifier.predict(X)
     >>> group_summary(accuracy_score, y_true, y_pred, sensitive_features=sex)
-    {'overall': 0.8445686557538159, 'by_group': {'female': 0.9256336459010306, 'male': 0.8044974759063791}}
+    {'overall': 0.844..., 'by_group': {'female': 0.925..., 'male': 0.804...}}
 
 Additionally, Fairlearn has lots of other standard metrics built-in, such as
 selection rate, i.e., the percentage of the population with label 1:
 
+.. doctest::
+
     >>> from fairlearn.metrics import selection_rate_group_summary
     >>> selection_rate_group_summary(y_true, y_pred, sensitive_features=sex)
-    {'overall': 0.1615429501550935, 'by_group': {'female': 0.059418809766966854, 'male': 0.21202386415787058}}
+    {'overall': 0.161..., 'by_group': {'female': 0.059..., 'male': 0.212...}}
 
 For a visual representation of the metrics try out the Fairlearn dashboard.
 While this page shows only screenshots, the actual dashboard is interactive.
 
+.. doctest::
+
     >>> from fairlearn.widget import FairlearnDashboard
     >>> FairlearnDashboard(sensitive_features=sex,
-                           sensitive_feature_names=['sex'],
-                           y_true=y_true,
-                           y_pred={"initial model": y_pred})
+    ...                    sensitive_feature_names=['sex'],
+    ...                    y_true=y_true,
+    ...                    y_pred={"initial model": y_pred}) # doctest: +SKIP
```

Comment:
why the dots?

Reply:
This isn't output, but code

Reply:
Those are indicating continuation lines.

Reply:
https://docs.python.org/3.8/library/doctest.html